.travis.yml

```diff
@@ -2,18 +2,14 @@ language: ruby
 rvm:
   - 2.2
   - 2.1
-  - 2.0
-matrix:
-  include:
-    - # Ruby 1.9
-      rvm: 1.9
-      env: JEKYLL_VERSION=2.0
 env:
   global:
     - NOKOGIRI_USE_SYSTEM_LIBRARIES=true
   matrix:
+    - JEKYLL_VERSION=3.3
     - JEKYLL_VERSION=3.0
     - JEKYLL_VERSION=2.5
+    - JEKYLL_VERSION=2.0
```

Comment:
Is there a reason to support Jekyll 2.0?
You could probably even drop Jekyll 2.5 at this point.